packages/utilities/moment-timezone-include/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# @bbc/moment-timezone-include Changelog
+
+<!-- prettier-ignore -->
+| Version | Description |
+| ------- | ----------- |
+| 1.0.0   | [PR#1521](https://github.com/BBC-News/psammead/pull/1521) Initial creation of package |
```

Comment:
This URL links back to the initial PR which was merged and then reverted. This should either be updated to this PR link, or at least make sure that everything's clearly linked in case there is an issue in the future